src/main/java/TaskManager.java

```diff
@@ -0,0 +1,43 @@
+public class TaskManager {
+    private static final int MAX_NUM_OF_TASKS = 100;
+    private static Task[] tasks = new Task[MAX_NUM_OF_TASKS];
+
+    public static void addTask(String type, String input) {
+        final String ADD_PHRASE_START = "added: ";
+        // add one because newTask has not been created yet
+        final String ADD_PHRASE_END =
+                "There are " + Integer.toString(Task.getTaskCount() + 1) + " task(s) in your list.";
+
+        Task newTask = Parser.parseTask(type, input);
+
+        // change to zero-index
+        tasks[Task.getTaskCount() - 1] = newTask;
+
+        System.out.println(ADD_PHRASE_START + newTask);
+        System.out.println(ADD_PHRASE_END);
+    }
+
+    public static void printList() {
+        for (int i = 0; i < Task.getTaskCount(); i += 1) {
+            System.out.println(Integer.toString(i + 1) + ". " + tasks[i]);
```

Comment:
No need Integer.toString to convert (i + 1) to string since println() will do it for you